docs/docusaurus/src/theme/DocSidebar/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
```

Comment:
Can we add a ticket to our backlog to look into making it so that the URL updates the correct anchor tag as you scroll the page?

Reply:
I also think we can add a test asserting the path in the DocSidebar is what's expected.

Reply:
I created the story here: [](https://greatexpectations.atlassian.net/jira/software/c/projects/DSB/boards/69?issueParent=26175&selectedIssue=DSB-723)